patches/server/0004-Paper-config-files.patch

```diff
@@ -1233,6 +1229,8 @@ index 0000000000000000000000000000000000000000..b2e961bbd33c6ecb7f049365b7aff6c5
 +                if (Files.isSymbolicLink(legacyConfig)) {
 +                    Files.delete(legacyConfig);
 +                }
++                Files.createFile(replacementFile);
++                Files.writeString(replacementFile, String.format(MOVED_NOTICE, configDir.toAbsolutePath()));
```

Comment:
Needs the notExists check

Reply:
I'm not sure if I did this right, but added notExists check.